src/main/java/nl/tudelft/cse1110/andy/execution/step/RunMetaTestsStep.java

```diff
@@ -120,7 +120,7 @@ private ResultBuilder runMetaTest(Context ctx, DirectoryConfiguration dirCfg, Fi
 
         ResultBuilder metaResult = new ResultBuilder(metaCtx, new GradeCalculator());
 
-        ExecutionFlow flow = ExecutionFlow.build(metaCtx, metaResult, new EmptyWriter());
+        ExecutionFlow flow = ExecutionFlow.build(metaCtx, metaResult, new EmptyWriter(), true);
```

Comment:
Oh here, I guess the security check should also be false in case we are running it without security checks, right?

Reply:
Maybe the info about whether it's running in secure mode should go to the Context class, so that you know which mode to run in the meta test.

Reply:
I think running it with security checks is a stronger assertion, and I think we want our tests working with the security manager. Also, I moved the boolean value to the `Context` class since this made indeed more sense.

Reply:
If we run with the security mode off, the meta steps should also run _without_ the security steps. (With this current version of the code, even if Andy is run without security checks, it will do the security checks for the meta tests.)

What's happening here is that we replace the code with the mutated version, and run "Andy on top of Andy". That's why you see a new _ExecutionFlow_ being instantiated.

Reply:
I understand it now, I just assumed this was only run in our local tests but I get it. I changed it and built the `ExecutionFlow` based on the boolean value of the `Context` passed to the method.